extension/xml.go

```diff
@@ -3,27 +3,9 @@ package extension
 import (
 	"encoding/xml"
 	"fmt"
-	"os"
 	"strings"
 )
 
-// GetS3ExtensionBucketHost returns the bucket to use for accessing the crx files
```

Comment:
Moved to utils, we'll probably get rid of the XML parsing code when C75 reaches stable.